src/riak_dt_map.erl

```diff
@@ -445,6 +455,11 @@ defer_remove(Clock, Ctx, Field, Deferred) ->
 
 %% @doc merge two `map()'s.
 -spec merge(map(), map()) -> map().
+merge({_LHSC, LHSE, LHSD}=LHS, {_RHSC, RHSE, RHSD}=RHS) when is_list(LHSE);
```

Comment:
I don't think so, I think it will be all or nothing, unless some crazyness is going on